ktor-client/ktor-client-curl/posix/src/io/ktor/client/engine/curl/internal/CurlMultiApiHandler.kt

```diff
@@ -217,10 +217,12 @@ internal class CurlMultiApiHandler : Closeable {
                         )
                     }
 
+                    val errorMessage = curl_easy_strerror(result)?.toKStringFromUtf8()
```

Comment:
will timeout exception produce an error here? if yes, can you add such a test?

Reply:
No, it will be handled before this line (line 214 actually) by checking for the particular timeout error code. I believe we already have timeout tests so no need to introduce new

Reply:
Are there any other errors that we can emulate to test this change?

Reply:
Yes, we could try to connect to a free port to emulate connection refused. On my machine I get the message `"Couldn't connect to server"` but I can't be sure that it won't change with difference systems/locales. I only can test that there are some characters at the corresponding position.